### ISSUE TITLE
`printType` always add `Concatenate` for ParamSpec even if there is no parameter to do Concatenate

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pyright#4932]
when you have this code

```python
from typing import Callable, ParamSpec
P = ParamSpec("P")
func: Callable[P, None]
```

calling `printType(typeOfFunc)` with 'PythonSyntax' flag returns `Callable[Concatenate[, P], None]` rather than `Callable[P, None]` 

looks like this code (https://github.com/microsoft/pyright/blob/main/packages/pyright-internal/src/analyzer/typePrinter.ts#L624) should handle the case where `paramTypes` is 0.

**Comment by erictraut** [comment]
Thanks for the bug report. Yes, your diagnosis was spot on. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.303, which I just published. The change will be included in a future version of pylance.
